rules.mak

```diff
@@ -55,6 +55,11 @@ ifdef DIRS
 	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) $(NL) )
 endif
 
+init:
+ifdef DIRS
+	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) init $(NL) )
```

Comment:
Which external projects depend on this definition?

Reply:
`make init` called from the home directory will perform a recursive `make init` call in the `third_party` directory. This change isn't actually needed - I'll remove it